button/button.css

```diff
@@ -0,0 +1,328 @@
+/* buttons */
+.buttons,
+.socials {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+/* normal buttons */
+.btn {
+  padding: 0.65rem;
+  border-radius: 0.25rem;
+  /* margin: 0.5rem; */
+}
+
+.btn-primary {
+  border: none;
+  color: var(--light-grey);
+  background: var(--purple);
+}
+
+.btn-secondary {
+  border: none;
+  color: var(--light-grey);
+  background: var(--textgrey);
+}
+
+.btn-success {
+  border: none;
+  color: var(--light-grey);
+  background: var(--darkgreen);
+}
+
+.btn-danger {
+  border: none;
+  color: var(--light-grey);
+  background: var(--darkred);
+}
+
+.btn-warning {
+  border: none;
+  color: var(--darkgrey);
+  background: var(--darkyellow);
+}
+
+.btn-info {
+  border: none;
+  color: var(--light-grey);
+  background: var(--darkblue);
+}
+
+.btn-light {
+  border: none;
+  color: var(--textgrey);
+  background: var(--light-grey);
+}
+
+.btn-dark {
+  border: none;
+  color: var(--light-grey);
+  background: var(--darkgrey);
+}
+
+.btn-link {
+  background: transparent;
+  border: none;
+  color: var(--darkblue);
+  text-decoration: underline;
+}
+
+/* actual buttons hover */
+.btn-primary:hover {
+  background: var(--darkpurple1);
+}
+
+.btn-secondary:hover {
+  background: var(--darkgrey);
+}
+
+.btn-success:hover {
+  background: var(--darkgreen1);
+}
+
+.btn-danger:hover {
+  background: var(--darkred1);
+}
+
+.btn-warning:hover {
+  background: var(--darkyellow1);
+}
+
+.btn-info:hover {
+  background: var(--darkblue1);
+}
+
+.btn-light:hover {
+  background: var(--light-grey);
+}
+
+.btn-dark:hover {
+  background: var(--darkgrey);
+}
+
+.btn-link:hover {
+  color: var(--darkblue1);
+}
+
+/* outline-buttons */
+.btn-outline-primary {
+  background: transparent;
+  color: var(--purple);
+  border: 1.5px solid;
+}
+
+.btn-outline-secondary {
+  background: transparent;
+  color: var(--textgrey);
+  border: 1.5px solid;
+}
+
+.btn-outline-success {
+  background: transparent;
+  color: var(--darkgreen);
+  border: 1.5px solid;
+}
+
+.btn-outline-danger {
+  background: transparent;
+  color: var(--darkred);
+  border: 1.5px solid;
+}
```

Comment:
Take the background and border property and either create a btn-outline class or apply them in all classes at once.